__tests__/integration/existing-collection.test.ts

```diff
@@ -0,0 +1,119 @@
+import { Client } from "../../src";
+import { endpoints } from "../../src";
+import { ServiceError } from "../../src";
+import { env } from "process";
+import { fql } from "../../src";
+
+const client = new Client({
+  endpoint: env["endpoint"] ? new URL(env["endpoint"]) : endpoints.local,
+  max_conns: 5,
+  secret: env["secret"] || "secret",
+});
```

Comment:
We've got a test helper function for this.

Note, it uses "FAUNA_SECRET" and "FAUNA_ENDPOINT" as the env vars.

```suggestion
import { getClient } from "../client";
import { ServiceError } from "../../src";
import { fql } from "../../src";

const client = getClient();
```